views/index.hbs

```diff
@@ -0,0 +1,33 @@
+<header class="jumbotron">
+  <div class="container">
+    <h1>Conformance Status Page</h1>
+    <p>Introduction text</p>
+  </div>
+</header>
+
+
+<section class="container">
+  <p>Main body section</p>
+  <table class="table table-hover datasets-table">
+    <thead>
+    <tr>
+      <th scope="col">col1</th>
```

Comment:
:-) If this was a normal review I'd request but as it's the first one in this repo I'm happy to just get a large bunch of code in so we can start tweaking this and other things - this table will change over time anyway.